src/routeTree.gen.ts

```diff
@@ -204,6 +210,13 @@ declare module '@tanstack/react-router' {
       preLoaderRoute: typeof UserUserAccessKeysImport
       parentRoute: typeof UserUserImport
     }
+    '/user/_user/destinations': {
+      id: '/user/_user/destinations'
+      path: '/destinations'
+      fullPath: '/user/destinations'
+      preLoaderRoute: typeof UserUserDestinationsImport
+      parentRoute: typeof UserUserImport
```

Comment:
Here naming is the same for all fields 👌🏻

Reply:
This code is generated by tanstack router